model/pipeline/src/main/proto/schema.proto

```diff
@@ -31,6 +31,11 @@ option go_package = "github.com/apache/beam/sdks/v2/go/pkg/beam/model/pipeline_v
 option java_package = "org.apache.beam.model.pipeline.v1";
 option java_outer_classname = "SchemaApi";
 
+message SchemaCoderPayload {
+  Schema schema = 1;
+  map<string, bytes> sdk_information = 2;
```

Comment:
I was thinking about this some more - will we ever actually get in a situation where multiple SDKs can attach information to a schema coder? How will that happen?

It seems like in practice just the producing SDK will attach sdk_information.

Reply:
Probably, why? Are you asking why it's a map? That was simply to make it simpler for the producing SDK to attach multiple items (in this case typeDescriptor, toRowFn, and fromRowFn)

Reply:
Ah yeah didn't realize you also had that reason for making it a map. I suppose then we don't need to be concerned about whether it's justified by multiple SDKs.

I'm still just curious how that might happen though.  Maybe @robertwb has thought about this.